plugins/statistics.koplugin/main.lua

```diff
@@ -409,20 +409,18 @@ function ReaderStatistics:onAddNote()
 end
 
 -- in case when screensaver starts
-function ReaderStatistics:onFlushSettings()
-    self:onSaveSettings()
+function ReaderStatistics:onSaveSettings()
+    self:saveSettings()
     self:exportToFile()
     self.current_period = 0
-    return true
```

Comment:
This line breaks event propagation in the input container of `ReaderUI` and `ReaderUI:onFlushSettings` won't be called.